receiver/dcgmreceiver/component_test.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build gpu
+// +build gpu
```

Comment:
Do we want to add a comment here that this file is basically the generated test with the build tag added? Ditto for `receiver/nvmlreceiver/component_test.go`?

Reply:
Added in [84e7ae5](https://github.com/GoogleCloudPlatform/opentelemetry-operations-collector/pull/208/commits/84e7ae5b24efd51bf0e6d707f0b7a0fa939e933a)